portability-transfer/src/main/java/org/dataportabilityproject/transfer/JobPollingService.java

```diff
@@ -125,19 +130,20 @@ private void claimJob(UUID jobId, KeyPair keyPair) throws IOException {
           (previous, updated) ->
               Preconditions.checkState(
                   previous.jobAuthorization().state() == JobAuthorization.State.CREDS_AVAILABLE));
-    } catch (IllegalStateException e) {
-      throw new IOException(
+    } catch (IllegalStateException | IOException e) {
```

Comment:
where is an IOException thrown?

Reply:
store.updateJob can throw an IOException.  Should an IOException be treated differently?

Reply:
ah ok, my bad